tests/bsim/bluetooth/ll/multiple_id/tests_scripts/multiple.sh

```diff
@@ -7,7 +7,7 @@ source ${ZEPHYR_BASE}/tests/bsim/sh_common.source
 # Multiple connection between two devices with multiple peripheral identity
 simulation_id="multiple"
 verbosity_level=2
-EXECUTE_TIMEOUT=2200
+EXECUTE_TIMEOUT=120
```

Comment:
"timeout" :)
(remember CI is slowish)

Reply:
The test finishes in 56 sec on my 6 year old laptop :)… 300 sec timeout is not enough… do I have to find the timeout trial-and-error on CI?

Reply:
(just as an example) If you made it say 50% shorter in your machine, then you could expect it also 50% shorter in CI, and that you could reduce the time by 50%.
Anyhow, if you look at the CI log, you can see it took 514 s in the last run in CI, so you better set it to 1500 s.

Reply:
I have it currently at 900, and passing on CI... do you want me to increase further to 1600?

Reply:
Yes please. Otherwise if CI is bogged down for some reason, it could timeout.